pyright/master/requirements.txt

```diff
@@ -0,0 +1,100 @@
+### Requirements: master
+#
+# This file defines the dependencies for the master pyright environment for the internal repo. This
+# is the environment that # pyright/pylance will use by default when running as a language server.
+#
+# Ideally, this would be the only pyright environment we need to define. This is not possible
+# because of dependency conflicts in between Dagster packages/examples. For ease of maintenance,
+# this file should maintain a complete list of Dagster packages, with packages excluded from the
+# enviroment commented out.
```

Comment:
Do we have a running list of known dependency conflicts? I'm sure that is in your head right now, and it would be useful to document.

Reply:
I could figure it out with some digging but it's surprisingly hard to figure out what the actual conflicts are. The way I arrived at the current partition is by repeatedly `pip install --dry-run` on our whole package list and eliminating until I could get a successful install.

Reply:
Got it. Not asking you to dig just to document what is resident in your brain cache.